test/projectile-test.el

```diff
@@ -51,33 +51,38 @@
     (should (equal (projectile-expand-root "./foo/bar") "/path/to/project/foo/bar"))))
 
 (ert-deftest projectile-test-ignored-directory-p ()
-  (noflet ((projectile-ignored-directories () '("/path/to/project/tmp")))
-    (should (projectile-ignored-directory-p "/path/to/project/tmp"))
-    (should-not (projectile-ignored-directory-p "/path/to/project/log"))))
+  (noflet ((projectile-ignored-directories () '("/path/to/project/tmp" "/path/to/project/t\\.*")))
+          (should (projectile-ignored-directory-p "/path/to/project/tmp"))
```

Comment:
You broke the indentation here. (likely because you didn't require `noflet` and Emacs didn't know how to indent it correctly.